### PR TITLE
PR for modal component!

### PR DESCRIPTION
**Pull request by adidoshi**
#### Hey! I create the modal component. Live preview link available below.
* (Primary modal).

### CODE REVIEW

**Review by manojkumar8400 — APPROVED** [review]
looks good to me.

File: components/modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  width: 100vw;
+  left: 0;
+  top: 0;
+  z-index: 99;
+  background-color: rgba(0, 0, 0, 0.2);
+}
+
+.modal {
+  background-color: white;
+  box-shadow: 0 10px 20px 1px rgba(6, 10, 36, 0.1),
```
Comment (mimansha-swarup) [inline components/modal/Modal.css:18]: make variable for box-shadow

File: components/modal/Modal.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  background-color: var(--primary-color);
+  color: white;
+  font-weight: bold;
+}
+
+.modal-footer-btn:hover {
+  background-color: var(--secondary-color);
+}
+
+.modal-footer-btn-outline {
```
Comment (mimansha-swarup) [inline components/modal/Modal.css:66]: suggestion:  you can use button from button components , also if possible

**Review by mimansha-swarup — COMMENTED** [review]
rest looks good

File: components/modal/Modal.css (hunk 1 above)
Comment (adidoshi) [reply components/modal/Modal.css:18]: Sure, I will make the changes thanks.

File: components/modal/Modal.css (hunk 2 above)
Comment (adidoshi) [reply components/modal/Modal.css:66]: Yeah, reusability. Sure, thanks.

**Pull request merged**